### PR TITLE
Added some new features and bug fixes

### PR DESCRIPTION
**Pull request by thooge**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Mark Zwart <mark.zwart@pobox.com>
+Przemyslaw Socha <sochap@gmail.com>
+Patrick Socha <sochap@gmail.com>
```
Comment (ppsx) [inline AUTHORS]: Hi.
Remove "Patrick Socha", please. It's just me (Przemyslaw Socha) using my middle name ;)
Cheers!

**Comment by Syco54645** [comment]
In testing out this code I find that it works better than the existing branch from mark-me however the touch buttons are very intermittent for me. I have a 7 inch tontec display. I suppose that would need added to the code base however the fall back support is not working as the buttons are not clickable. I can provide resolution details for my monitor if you'd like (I think it was 720x525 but do not have the thing in front of me right now) however the touch buttons should work in fallback no?

I should also say that the buttons work occasionally. Not sure if it is the hit area or the driver that is being messed up. I can create a fork and prepare a commit if you'd like.

**Comment by ppsx** [comment]
Please provide exact resoultion.
Nevertheless it can take some time to introduce proper configuration for higher resolutions.
For now we don't have any mechanism to automatically rearrange the size and positions of screen elements (such as buttons, labels, etc.). However you can play with some settings in settings.py file.

**Comment by thooge** [comment]
I'm currently working on changes for my Raspberry 7" display.
Changes will appear in the near future ;-) Stay tuned.

**Comment by Syco54645** [comment]
> Please provide exact resoultion.
> mode "720x576"
>     geometry 720 576 720 576 16
>     timings 0 0 0 0 0 0 0
>     rgba 5/11,6/5,5/0,0/16
> endmode
> Bus 001 Device 005: ID 0eef:0001 D-WAV Scientific Co., Ltd eGalax TouchScreen

**Pull request closed without merging**